TestApp/app/screens/AppCenterScreen.js

```diff
@@ -233,6 +235,10 @@ export default class AppCenterScreen extends Component {
                   value: 'userId',
                   onChange: async (userId) => {
                     this.setState({ userId });
+                  },
+                  onSubmit: async () => {
+                    // TODO: 1DS setUserId API allows null but not empty string as userId
```

Comment:
It's by design not a "todo" for 1DS.
```suggestion
                    // 1DS setUserId API allows null but not empty string as userId
```